ehrapy/anndata/anndata_ext.py

```diff
@@ -467,6 +469,26 @@ def _update_uns(
         return all_moved_num_columns, list(all_moved_non_num_columns), None
 
 
+def _detect_binary_columns(df: pd.DataFrame, numerical_columns: list[str]) -> list[str]:
+    """
+    Detect all columns that contain only 0 and 1 (besides NaNs)
```

Comment:
BTW, the first docstring sentence has to be after the triple quotes and must end with a `.`